### PR TITLE
Integration test

### PR DESCRIPTION
**Pull request by vshanthe**
## 📝 Description
Added Integration test to Account and Databases section 

**What does this PR do and why is this change necessary?**
Test coverage across all the section 

## ✔️ How to Test
`make INTEGRATION_TEST_PATH=account testint`

`make INTEGRATION_TEST_PATH=database testint`

`make INTEGRATION_TEST_PATH=beta testint`

## 📷 Preview

**If applicable, include a screenshot or code snippet of this change. Otherwise, please remove this section.**

### CODE REVIEW

File: tests/integration/beta/test_beta_program.py
Comment (zliang-akamai) [inline tests/integration/beta/test_beta_program.py:14]: There can be a case that in some period of time, there might not be any available beta program. So I think we should skip this line if `len(lines[1].split(",")) == 0` and also skip the view tests below

File: tests/integration/database/test_database.py
Comment (zliang-akamai) [inline tests/integration/database/test_database.py:1]: The current generation DBaaS has been discontinued, and we might want to hold off on these tests.

File: tests/integration/account/test_account.py
Comment (ykim-1) [inline tests/integration/account/test_account.py:30]: Just a general comment here, if we are only comparing headers for all the tests might be cleaner to create one helper function that can be called when asserting

File: tests/integration/account/test_account.py
Comment (ykim-1) [inline tests/integration/account/test_account.py:181]: Is `user_id` this required for assertion?

File: tests/integration/account/test_account.py
Comment (vshanthe) [reply tests/integration/account/test_account.py:186]: yes for the test_user_view command requires the user_id

File: tests/integration/database/test_database.py
Comment (vshanthe) [reply tests/integration/database/test_database.py:1]: so we don't need these tests ?

File: tests/integration/account/test_account.py
Comment (vshanthe) [reply tests/integration/account/test_account.py:173]: This will be used to pass in the next method to view command

**Comment by ykim-1** [comment]
database test cases are failing locally for me

**Comment by vshanthe** [comment]
> database test cases are failing locally for me

Screenshot 2024-03-26 at 9 49 17 AM

**Comment by vshanthe** [comment]
> > database test cases are failing locally for me
> > Is any particular test failing or complete , i don't see any failure while running 
> Screenshot 2024-03-26 at 9 49 17 AM

**Comment by ykim-1** [comment]
> > > database test cases are failing locally for me
> > > Is any particular test failing or complete , i don't see any failure while running
> > > Screenshot 2024-03-26 at 9 49 17 AM

I am getting header not found in assertion on all of the tests. Do we have to edit anything for the account?

```
args = ['linode-cli', 'databases', 'types', '--text', '--delimiter=,']

    def exec_test_command(args: List[str]):
        process = subprocess.run(args, stdout=subprocess.PIPE)
>       assert process.returncode == 0
E       AssertionError

tests/integration/helpers.py:33: AssertionError
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- Captured stderr call --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Request failed: 404
errors
field,reason
,Not found
```

File: tests/integration/database/test_database.py
Comment (zliang-akamai) [reply tests/integration/database/test_database.py:1]: Yes, I think we can exclude them for now.

**Comment by ykim-1** [comment]
Giving approval for now. It would be good to have one successful e2e build including all the newly added tests on this PR before merging 👍

**Review by ykim-1 — APPROVED** [review]

**Review by zliang-akamai — APPROVED** [review]
All passed locally. Nice work!

**Pull request merged**
